src/main/java/org/fundacionjala/sfdc/pageobjects/Main.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import java.util.Properties;
+import org.fundacionjala.sfdc.commons.DriverManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.commons.DriverManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)